thirdeye/thirdeye-frontend/config/environment.js

```diff
@@ -23,6 +23,8 @@ module.exports = function(environment) {
       directory: 'app/mirage'
     },
 
+    https_only: false,
```

Comment:
Will this disable https by default?

Reply:
disable this by default in the open-source project otherwise opensource version will return an error when opening ThirdEye. I will config it in thirdeye-configs.

Reply:
I see. Thanks.